packages/combobox/src/Combobox/Combobox.spec.tsx

```diff
@@ -763,7 +765,7 @@ describe('packages/combobox', () => {
             });
             const carrotChip = queryChipsByName('Carrot');
             const carrotChipButton = carrotChip!.querySelector('button');
-            userEvent.click(carrotChipButton!);
+            expect(() => userEvent.click(carrotChipButton!)).toThrow();
```

Comment:
Do we want to expect this user event to fail?

Reply:
yeah, with the latest version of `userEvents`, elements with `pointer-events: none` are not clickable so it should throw.

Reply:
https://github.com/testing-library/user-event/pull/647